HIP/hip-1064.md

```diff
@@ -0,0 +1,165 @@
+---
+hip: 1064
+title: Daily Rewards For Active Nodes
+author: Neeharika Sompalli <@Neeharika-Sompalli>
+working-group: Leemon Baird (@lbaird), Richard Bair <@rbair23>, Michael Tinker <@tinker-michaelj>
+requested-by: Hashgraph
+type: Standards Track
+category: Core
+needs-council-approval: Yes
+status: Draft
+last-call-date-time: 2024-04-08T07:00:00Z
+created: 2024-10-17
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/1063
+updated: 2024-10-17
+---
+
+## Abstract
+
+This HIP proposes a reward mechanism that will incentivize nodes to remain active on the network. The definition of active node is described below. The minimum reward per day for nodes will be a configurable property decided by the council. This minimum reward may be adjusted based on fees already paid throughout the day.
+
+## Motivation
+
+The goal is to give a strong incentive for node operators to keep them active and properly configured throughout every staking period.
+
+## Rationale
+
+Clearly defining the criteria for node activity, we can ensure that rewards are distributed fairly, incentivizing nodes to remain active and contribute to the network’s stability and growth.
+
+### Terminology
+
+In the Hashgraph consensus algorithm, nodes communicate by sharing information through events. An event contains some transactions.
+
+In each round, each node creates a *witness* which is an event created in that round that does not have any descendants from the same node in that round. A *judge* is a specific type of witness event that gains importance because it is known and agreed upon by a majority of the network.
+
+This HIP defines a node as *active* if it creates a judge in a significant fraction of rounds during a staking period, for example 80%. Any node that is trustworthy and submits events will create judges. If the node doesn’t submit events or when other nodes don’t build on this node’s events ,  the node will not create judges.
+
+**User Stories**
+
+The implementation will be tested under various scenarios, including:
+
+- As a node operator, I will expect to get a minimum node reward when the node is active for atleast `nodes.activeRoundsPercent`  rounds
```

Comment:
```suggestion
- As a node operator, I will expect to get a minimum node reward when the node is active for at least `nodes.activeRoundsPercent`  rounds
```

Some other lines also should have "at least" as two words instead of 1

Reply:
Yes changed them . thanks Richard